paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -120,15 +167,16 @@ class FCMKLDNNHandler
 
     // ReLU from "fc_fuse_pass"
     if (ctx.Attr<std::string>("activation_type") == "relu") {
+      post_operations.append_eltwise(dnnl::algorithm::eltwise_relu, 0.0f, 0.0f);
       post_operations.append_eltwise(
-          activation_scale, dnnl::algorithm::eltwise_relu, 0.0f, 0.0f);
+          dnnl::algorithm::eltwise_linear, activation_scale, 0.0f);
```

Comment:
Why not use DNNL_ARG_DST SCALE for activation_scale since it will be applied after activation.